### PR TITLE
[DE] Add `Music theory/Syncopation`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Notes

Here are some useful tutorials about the German terms used in music theory. I recommend taking a look at these before reviewing the article.

- [Wikipedia article "Beat"](https://de.wikipedia.org/wiki/Beat_(Rhythmus))
- [Wikipedia article "Metrum"](https://de.wikipedia.org/wiki/Metrum_(Musik))
- [Clavio.de thread](https://www.clavio.de/threads/off-onbeat-vs-up-downbeat.2428/)
- [Wikipedia article "Synkope"](https://de.wikipedia.org/wiki/Synkope_(Musik))
- [YouTube playlist "Musikalische Begriffe einfach erklärt"](https://www.youtube.com/watch?v=NtCIB5DfcTg&list=PLUY8A0N9d78CokHmbXzjj5MTsGexkSdVX&index=4)

The other translations for the music theory articles are still work in progress. However, I would like to propose the following translation guidelines:

- Beat - Takt
- Time signature - Taktart (see #10155)
- Measure - Metrum (Grundschlag)
- Rhythm - Rhythmus
- Layer - ("Textur" maybe; nothing decided yet; still need to comprehend the article's topic)

The terms in brackets will be added to the respective article as an alternative (e.g. `also known as ...`).

Of course, I'm aware that these translations should not be fixed and depending on the context can be even wrong (like [described here](https://github.com/ppy/osu-wiki/pull/10155#issuecomment-1762884498)). Though as a general direction, I wanted to mention this here.

Regarding the translation, I have a few notes to make.

- `red and blue ticks`: I think in the past, I used the word `Tick` for translation. In this article, I used `Striche` instead, because I think that is okay too.
- `onbeat`: I don't think onbeat is actually used in German, however, I have no idea if `Grundschlag` or `Takt` is better as an alternative.
- `drum kicks` and `percussion`: I'm not sure if these are translated correctly. Both seem to be something drum-like to me in this context.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by FreezyLemon** [comment]
I agree with `time signature` = `Taktart` and `rhythm` = `Rhythmus`. I'd have to read the other english articles to get an idea about `layer`. Small aside: Why not translate the basics first (beat, rhythm, layer) instead of jumping directly into syncopation?

Anyways, my thoughts regarding the other TL suggestions:
The first sentences in the `Beat` Wikipedia article state that `Beat ist die englische Bezeichnung für den Taktschlag in einem Musikstück, [...]. In der Popmusik wird mit „Beat“ im weiteren Sinn ein ausgeprägter Rhythmus bezeichnet.` The english osu-wiki article seems to agree with this, so I'd go with `Taktschlag` (or the synonym `Grundschlag`). The word `Beat` can probably also be used in German (e.g. when referring to it in other articles), but for the explanation of musical theory it's better to use the technical term, since `Beat` is also used colloquially. The secondary meaning of a `Beat` describing a song's rhythm in a wider sense can be ignored here IMHO.

Looking at the english osu-wiki article for `measure`, the translation `Takt` (not `Taktart`[^note-takt]) seems obvious: `Das Lied hat vier Grundschläge in einem Takt, da es in der Taktart 4/4 geschrieben wurde.`

[^note-takt]: `Takt` is sometimes used to refer to the `Taktart` (`Das Lied ist im Dreivierteltakt geschrieben` = `Das Lied ist in der Taktart 3/4 geschrieben`), but never the other way around.

**Review by The-Last-Cookie — COMMENTED** [review]
I have taken another look at this article and I think it's fine to review (I'm referring to the discussion on discord regarding the translation of musical terms).

Please keep in mind my translation notes (e.g. `onbeat` and `tick`) while reviewing.

File: wiki/Music_theory/Syncopation/de.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+---
+
+# Synkope
+
+Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```
Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:7]: Should we distinguish `(eine) Synkope` and `Synkope/Synkopieren`? According to the German sources, `eine Synkope`, which is what you're using here, only describes one instance of this irregular rhythm construct. The english osu-wiki article refers to `Syncopation` (not `a syncopation` (singular)), which means the process of using syncopations regularly throughout the song. To be technically correct here, I think we'd need to use `Synkopieren` (a `substantiviertes Verb`), which doesn't sound great though. Something like this maybe:

```suggestion
**Synkopieren** beschreibt die regelmäßige rhythmische Betonung von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```

Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:7]:
```suggestion
Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```

Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:7]:
```suggestion
Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.
```

File: wiki/Music_theory/Syncopation/de.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl im Onbeat als auch im Offbeat liegen.
```
Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:9]: I think we should avoid `Onbeat`:

```suggestion
Das [Insane-Level von *Chasers - Lost* erstellt von fanzhen0019](https://osu.ppy.sh/beatmapsets/151114#osu/372628) enthält als einfaches Beispiel [Slider](/wiki/Gameplay/Hit_object/Slider)-Patterns mit [Sliderköpfen](/wiki/Gameplay/Hit_object/Slider/Sliderhead), die sowohl auf dem Beat als auch auf dem Offbeat liegen.
```

File: wiki/Music_theory/Syncopation/de.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Einige Slider-Patterns, wie zum Beispiel das allererste, folgen der Synth-Melodie, die mit den Trommelschlägen synkopiert ist, und landen auf roten Strichen auf der Zeitleiste:
+
+![Beispiel eines Sliderskopfes, der einem synkopischen Rhythmus folgt](img/syncopated-sliders.png)
```
Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:13]:
```suggestion
![Beispiel eines Sliderkopfes, der einem synkopischen Rhythmus folgt](img/syncopated-sliders.png)
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
Not sure about that `Synkope`/`Synkopieren` differentiation. That's for you to decide

File: wiki/Music_theory/Syncopation/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Music_theory/Syncopation/de.md:9]: My intention was to avoid `Onbeat` as well.

File: wiki/Music_theory/Syncopation/de.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+---
+
+# Synkope
+
+**Synkopen** sind die Akzentuierung von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.
```
Comment (FreezyLemon) [inline wiki/Music_theory/Syncopation/de.md:7]:
```suggestion
**Synkopen** sind Akzentuierungen von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks deutet ein [rhythmisches](/wiki/Music_theory/Rhythm) Muster aus starken und schwachen Beats an, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen gesetzt werden. Beim [Beatmapping](/wiki/Beatmapping) kann man Synkopen normalerweise daran erkennen, dass [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, also zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste platziert sind.
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
looks good, only a small thing

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**